<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmReport</class>
 <widget class="QDialog" name="ConfirmReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>293</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>多点远程超声 - 审核报告</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>审核意见：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="confirm_msg_text_edit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>421</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>（注意：报告审核通过后将不能再做修改。）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>250</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reject_button">
   <property name="geometry">
    <rect>
     <x>238</x>
     <y>250</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>审核未通过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pass_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>审核通过</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>